config/default.yml

```
@@ -4700,6 +4700,16 @@ Style/UnlessElse:
   Enabled: true
   VersionAdded: '0.9'
 
+Style/UnlessLogicalOperators:
+  Description: >-
+                 Checks for use of logical operators in an unless condition.
+  Enabled: pending
```

It seems too strict to enable this rule by default in future (or `NewCops: enable`). So I think this cop should be disabled by default.

```suggestion
  Enabled: false
```

---

Thanks. I was hoping to get feedback on this.